### PR TITLE
In test_window test_info_dialog is added

### PR DESCRIPTION
**Pull request by GrayR00t**
In test_window code is added to test info_dialog function of window

automatic testing of info_dialog function of window.py

## PR Checklist:

- [ x] All new features have been tested
- [ x] All new features have been documented
- [ x] I have read the **CONTRIBUTING.md** file
- [ x] I will abide by the code of conduct

### CODE REVIEW

File: src/core/tests/test_window.py [hunk 1]
```diff
@@ -64,3 +64,10 @@ def test_on_close(self):
         with patch.object(self.window, '_impl'):
             self.window.on_close()
             self.window._impl.on_close.assert_called_once_with()
+
+    def test_info_dialog(self):
+        title  ='checking_info_dialog'
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 14: (E221) multiple spaces before operator

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 17: (E225) missing whitespace around operator

File: src/core/tests/test_window.py [hunk 2]
```diff
@@ -64,3 +64,10 @@ def test_on_close(self):
...
+        message = 'completed'
+        with patch.object(self.window ,'_impl'):
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 38: (E203) whitespace before ','

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 39: (E231) missing whitespace after ','

File: src/core/tests/test_window.py [hunk 3]
```diff
@@ -64,3 +64,10 @@ def test_on_close(self):
...
+            self.window.info_dialog(title ,message)
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 42: (E203) whitespace before ','

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 43: (E231) missing whitespace after ','

File: src/core/tests/test_window.py [hunk 4]
```diff
@@ -64,3 +64,10 @@ def test_on_close(self):
...
+            self.window._impl.info_dialog.assert_called_once_with(title ,message)
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 1: (W391) blank line at end of file

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 72: (E203) whitespace before ','

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 73: (E231) missing whitespace after ','

File: src/core/tests/test_window.py [hunk 5]
```diff
@@ -64,3 +64,11 @@ def test_on_close(self):
         with patch.object(self.window, '_impl'):
             self.window.on_close()
             self.window._impl.on_close.assert_called_once_with()
+
+    def test_info_dialog(self):
+        # title and message for testing
+        title = 'checking_info_dialog'
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 39: (W291) trailing whitespace

File: src/core/tests/test_window.py [hunk 6]
```diff
@@ -64,3 +64,11 @@ def test_on_close(self):
         with patch.object(self.window, '_impl'):
             self.window.on_close()
             self.window._impl.on_close.assert_called_once_with()
+
+    def test_info_dialog(self):
+        # title and message for testing
+        title = 'checking_info_dialog' 
+        message = 'completed'
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 30: (W291) trailing whitespace

File: src/core/tests/test_window.py [hunk 7]
```diff
@@ -64,3 +64,11 @@ def test_on_close(self):
...
             self.window.on_close()
             self.window._impl.on_close.assert_called_once_with()
+
+    def test_info_dialog(self):
+        # title and message for testing
+        title = 'checking_info_dialog' 
+        message = 'completed' 
+        with patch.object(self.window, '_impl'):
+            self.window.info_dialog(title, message)
+            self.window._impl.info_dialog.assert_called_once_with(title, message)
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 1: (W391) blank line at end of file

File: src/core/tests/test_window.py [hunk 8]
```diff
@@ -64,3 +64,12 @@ def test_on_close(self):
...
             self.window._impl.on_close.assert_called_once_with()
+
+    def test_info_dialog(self):
+        # title and message for testing
+        title = 'checking_info_dialog'
+        message = 'completed'
+        with patch.object(self.window, '_impl'):
+            self.window.info_dialog(title, message)
+            self.window._impl.info_dialog.assert_called_once_with(title, message)
+
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 1: (W391) blank line at end of file

Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 1: (W293) blank line contains whitespace

File: src/core/tests/test_window.py [hunk 9]
```diff
@@ -6,15 +8,18 @@
 
 
 class TestWindow(TestCase):
+
     def setUp(self):
         super().setUp()
         self.window = toga.Window(factory=toga_dummy.factory)
 
     def test_widget_created(self):
         id = self.window.id
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 9: (F841) local variable 'id' is assigned to but never used

File: src/core/tests/test_window.py [hunk 10]
```diff
@@ -64,3 +76,14 @@ def test_on_close(self):
...
+    def test_info_dialog(self):
+
+        # title and message for testing
+
+        title = 'checking_info_dialog'
+        message = 'completed'
+        with patch.object(self.window, '_impl'):
+            self.window.info_dialog(title, message)
+            self.window._impl.info_dialog.assert_called_once_with(title,
+                    message)
```
Comment (brutusthebee) [inline src/core/tests/test_window.py]: At column 21: (E128) continuation line under-indented for visual indent

File: src/core/tests/test_window.py [hunk 11]
```diff
@@ -8,18 +6,15 @@
...
         id = self.window.id
         self.assertIsNotNone(self.window.id)
-        app = toga.App('test_name', 'id.app',
-                       factory=toga_dummy.factory)
-        new_app = toga.App('error_name', 'id.error',
-                           factory=toga_dummy.factory)
+        app = toga.App('test_name', 'id.app', factory=toga_dummy.factory)
+        new_app = toga.App('error_name', 'id.error', factory=toga_dummy.factory)
         self.window.app = app
         with self.assertRaises(Exception):
```
Comment (Aareon) [inline src/core/tests/test_window.py]: Specifically what exception might this be expecting? I don't know how I feel about using a `Exception` here.

File: src/core/tests/test_window.py [hunk 12]
```diff
@@ -8,18 +6,15 @@
 
 
 class TestWindow(TestCase):
-
     def setUp(self):
         super().setUp()
         self.window = toga.Window(factory=toga_dummy.factory)
 
     def test_widget_created(self):
         id = self.window.id
```
Comment (Aareon) [inline src/core/tests/test_window.py]: From @brutusthebee
> At column 9: (F841) local variable 'id' is assigned to but never used

**Comment by phildini** [comment]
Hi there! It looks like this PR might be dead, so we're closing it for now. Feel free to re-open it if you'd like to continue. 😄

**Pull request closed without merging**